torchao/experimental/quant_api.py

```diff
@@ -482,6 +492,104 @@ def quantize(self, model: nn.Module) -> nn.Module:
         return model
 
 
+_intx_granularity = Union[PerGroup, PerRow]
+
+
+def int8_dynamic_activation_intx_weight_v2(
```

Comment:
is there a significant diffrence between this and int8_dynamic_activation_intx_weight?